testing/ios/IosUnitTests/README.md

```diff
@@ -1,16 +1,39 @@
 # iOS Unit Tests
 
-These are the unit tests for iOS engine.  They can be executed locally and are
+These are the unit tests for iOS engine. They can be executed locally and are
 also run in LUCI builds.
 
-## Running Tests
+## Running Tests from command line
 
-To build and run the iOS tests, run the following script from the src directory:
+To build and run the iOS tests, use the `run_tests` script from the `src` directory.
 
 ```sh
 flutter/testing/run_tests.py --type=objc
 ```
 
+And if you're on Apple Silicon:
+
+```sh
+./flutter/testing/run_tests.py \
+  --type=objc \
+  --variant=host_debug_unopt_arm64 \
+  --ios=ios_debug_sim_unopt_arm64
```

Comment:
Shouldn't this be `--ios-variant`?
```suggestion
  --ios-variant ios_debug_sim_unopt_arm64
```

Reply:
That's right! But there's no difference in behavior between:

```
./flutter/testing/run_tests.py \
  --type=objc \
  --variant=host_debug_unopt_arm64 \
  --ios=ios_debug_sim_unopt_arm64
```

and

```
$ ./flutter/testing/run_tests.py \
  --type=objc \
  --variant=host_debug_unopt_arm64 \
  --ios-variant=ios_debug_sim_unopt_arm64
```

Apparently Python's argparser sees that the only option that starts with `ios` is `ios-variant` and it all goes well thanks to this.

Anyway thanks for the suggestion, fixed.